networkx/algorithms/community/tests/test_divisive.py

```diff
@@ -64,10 +63,9 @@ def test_edge_current_flow_betweenness_partition():
         assert s in C
 
     C = nx.community.edge_current_flow_betweenness_partition(G, 4)
-    answer = [{1, 2}, {4, 5, 6}, {3}, {0}]
-    assert len(C) == len(answer)
-    for s in answer:
-        assert s in C
+    answers = [[{1, 2}, {4, 5, 6}, {3}, {0}], [{0, 1, 2}, {5, 6}, {3}, {4}]]
```

Comment:
Is there any special reason why the partitions on the barbells *have* to contain specific nodes? For example, isn't `[{0, 2}, {4, 5, 6}, {3}, {1}]` (swapping nodes 0 and 1) just as valid as the first listed option? If so, then it might be good to include as a possible answer for completeness' sake. Applying the same pattern (swapping nodes in the other bell) would give more equally valid partitionings.

Reply:
instead of adding all of the valid solutions we can instead try setting the kind of random algo we want to use for the OS(something like this: https://stackoverflow.com/questions/47199415/is-set-seed-consistent-over-different-versions-of-r-and-ubuntu/56381613#56381613, i was trying to search for a `random.seed` like thing for OSs). But, I'm not sure if this can be in the github workflow. What do you think?